packages/react-native-cli/src/commands/AutomateSymbolicationCommand.ts

```diff
@@ -56,6 +78,32 @@ export default async function run (projectRoot: string, urls: OnPremiseUrls): Pr
   }
 }
 
+async function installBugsnagCliPackage (projectRoot: string): Promise<void> {
+  const alreadyInstalled = await detectInstalled('@bugsnag/cli', projectRoot)
+
+  if (alreadyInstalled) {
+    logger.warn('@bugsnag/cli is already installed, skipping')
+    return
+  }
+
+  logger.info(alreadyInstalled)
+
+  logger.info('Adding @bugsnag/cli dependency')
+
+  const packageManager = await guessPackageManager(projectRoot)
+
+  const { version } = await prompts({
+    type: 'text',
+    name: 'version',
+    message: 'If you want the latest version of @bugsnag/cli hit enter, otherwise type the version you want',
+    initial: 'latest'
+  }, { onCancel })
+
+  await install(packageManager, '@bugsnag/cli', version, true, projectRoot)
+
+  logger.success('@bugsnag/cli dependency is installed')
+}
+
 async function installJavaScriptPackage (projectRoot: string): Promise<void> {
```

Comment:
We agreed that we would insert the command into the package.json as part of this init operation, under a prompt (e.g. "Do you want to add an NPM task to your package.json to upload Android source maps?")